scripts/action-handler.js

```diff
@@ -1266,54 +1263,42 @@ Hooks.once('tokenActionHudCoreApiReady', async (coreModule) => {
          * @returns {object}
          */
         #getItemInfo (item) {
-            const quantityData = this.#getQuantityData(item)
-            const usesData = this.#getUsesData(item)
-            const consumeData = this.#getConsumeData(item)
+            const info1 = item.type !== 'spell' ? this.#getQuantityData(item) : this.displaySpellInfo ? this.#getSpellInfo(item) : null
```

Comment:
Move `this.displaySpellInfo` check to `getSpellInfo`

Reply:
Good for consistency. I'll merge it with the check if it the spell even has concentration, so it is just one if statement in that function.